src/renderer/shared/ui/Buttons/Button/Button.tsx

```diff
@@ -43,6 +43,7 @@ export const Button = forwardRef<HTMLButtonElement, PropsWithChildren<Props>>(
       onPointerMove,
       onPointerLeave,
       testId = 'Button',
+      ...rest
```

Comment:
Is required by Radix - [doc](https://www.radix-ui.com/primitives/docs/guides/composition#composing-with-your-own-react-components)